src/appleseed.shaders/src/alshaders/alSurface.osl

```diff
@@ -468,10 +468,11 @@ shader alSurface
             transmissionNormal,
             vector(0),
             transmissionColor,
-            color(0), // transmission only.
+            color(0),                       // Transmission only.
```

Comment:
Our convention for end-of-line comments is no capital, no period:

    // transmission only